nais.yaml

```diff
@@ -22,10 +22,10 @@ spec:
     enabled: {{prometheusEnabled}}
     path: /prometheus
   liveness:
-    path: /sosialhjelp/internal/isAlive
```

Comment:
/internal er sperret slik at det ikke kan nåes fra www.nav.no. 
Derfor har vi alle helseskjekker, metrikker etc. bak /internal. 
Nå er det ikke krise om helsesjekker fra frontend vises ut, men om målet er å ha det så likt som mulig så blir det mer likt de andre appene om det fortsatt ligger bak /internal

Reply:
For at next skal oppfatte dette som et api-endepunkt så må det ligge under `/api`. Kan eventuelt endre begge appene til å bruke `/api/internal/isAlive|isReady` etterhvert hvis vi ønsker det likt som backend. :)